packages/registry/src/types/index.ts

```diff
@@ -1,14 +1,26 @@
 export { SettingsConfig } from './Settings';
 export { BlocksConfig } from './Blocks';
+export { ViewsConfig } from './Views';
+export { WidgetsConfig } from './Widgets';
 
-export type ExperimentalConfig = {
+export type AddonReducersConfig = {
+  [key: string]: Function;
+};
+
+export type AddonRoutesConfig = {
+  path: string;
+  exact: boolean;
+  component: React.ComponentType;
+}[];
+
+export type SlotsConfig = {
   [key: string]: unknown;
 };
-export type ViewsConfig = {};
-export type WidgetsConfig = {};
-export type AddonReducersConfig = {};
-export type AddonRoutesConfig = {};
-export type SlotsConfig = {};
+
 export type ComponentsConfig = {
```

Comment:
FYI you can write this more readable as: 

`export type ComponentsConfig = Record<string, { component: React.ComponentType }>;`

As far as I know, the used syntax is prefered if you extend this types with other attributes, eg.:

```
export type ComponentsConfig = {
   [key: string]: { component: React.ComponentType };
   name2: { component: Function; };
   name3: { component: string; };
};
```